### PR TITLE
Added Service Accounts page to settings

### LINKED ISSUE #53: Service Accounts
Need to integrate [service accounts api from madmin](https://github.com/minio/mc/pull/3105)

- [x] API to List Service Accounts for a user
- [x] API to Add Service Account for a user. (Credentials should be auto generated)
- [x] API to Delete Service Account for a user
- [x] UI Screen to list Service accounts
- [x] UI Dialog to Add a new Service Account
- [x] UI Dialog to Delete a Service Account

### PR DESCRIPTION
**Pull request by bexsoft**
fixes #53 

## What does this do?

Added service accounts page to settings module

## How to test

- Create an account with permissions to see service accounts or add this fragment of code into line 350 of Console.tsx:

```
<Route
                    exact
                    path="/service_accounts"
                    component={ServiceAccounts}
                  />
```

- Go into Configurations Lists > service_accounts > Edit Icon
- Click on create service account & write a valid policy
- Click on create button, the policy should appear in the list
- Click on the delete button of the recently created policy, the policy should be deleted

Screen Shot 2020-05-18 at 11 37 52 PM
Screen Shot 2020-05-18 at 11 37 34 PM
Screen Shot 2020-05-18 at 11 37 28 PM
Screen Shot 2020-05-18 at 11 37 15 PM

### CODE REVIEW

**Comment by cesnietor** [comment]
I'm also trying to test it, there is no permission needed to See service Accounts, but I can't see the page load. And even having admin:* is not showing also.

**Comment by bexsoft** [comment]
> I'm also trying to test it, there is no permission needed to See service Accounts, but I can't see the page load. And even having admin:* is not showing also.

Just fixed this issue, page is now fully working

File: pkg/acl/endpoints.go [hunk 1]
```diff
@@ -179,19 +180,34 @@ var bucketsActionSet = ConfigurationActionSet{
...
+var serviceAccountsActionSet = ConfigurationActionSet{
+	actionTypes: iampolicy.NewActionSet(
+		iampolicy.AllAdminActions,
+	),
+	actions: iampolicy.NewActionSet(
+		iampolicy.ListUsersAdminAction,
+		iampolicy.CreateUserAdminAction,
+		iampolicy.DeleteUserAdminAction,
+		iampolicy.GetUserAdminAction,
+		iampolicy.ConfigUpdateAdminAction,
```
Comment (cesnietor) [inline pkg/acl/endpoints.go:193]: these policies are not for Service Accounts are for users, there is actually no policy needed to create/list/delete Service Account. If you just create a user and set a simple policy to read one bucket that user should be able to create service accounts.

Comment (bexsoft) [reply pkg/acl/endpoints.go]: As we talked, Policies have been removed for this page

**Pull request merged**